File: java/e-java/chegg-13/src/Customer.java

public class Customer {
	private int id;
	private String name;
	
	/**
	 * 
	 * @param id customer id
	 * @param name customer name
	 */
	public Customer(int id, String name) {
		this.id = id;
		this.name = name;
	}
	
	/**
	 * 
	 * @return name of the customer
	 */
	public String getName() {
		return this.name;
	}
	
	/**
	 * 
	 * @return id of the customer
	 */
	public int getId() {
		return this.id;
	}
	
	/**
	 * @return customer id and customer name as a string 
	 */
	public String toString() {
		String res;
		res = this.id +": "+this.name;
		return res;
	}
}

File: c++/q42/Customer.cpp
#include<iostream>
using namespace std;

class Customer{
	string name;
	string operation;
	double money;
	int age;
	
	public:
		Customer(string name, string operation, double money, int age){
			this->setAge(age);
			this->setName(name);
			this->setMoney(money);
			this->setOperation(operation);
		}
		
		/**
		* function to set name of customer
		* @param name of customer
		*/
		void setName(string name){
			this->name = name;
		}
		
		/**
		* function to set operation of customer
		* @param operaion (Deposit/ Withdraw)
		*/
		void setOperation(string operation){
			this->operation = operation;
		}
		
		/**
		* function to set mmoney of customer 
		* @param money
		*/
		void setMoney(double money){
			this->money = money;
		}
		
		/**
		* function to set age of customer
		* @param age of customer
		*/
		void setAge(int age){
			this->age = age;
		}
		
		/**
		* @return name of customer
		*/
		string getName(){
			return this->name;
		}
		
		/**
		* return operation of customer
		*/
		string getOperation(){
			return this->operation;
		}
		
		/**
		* @return money of customer
		*/
		double getMoney(){
			return this->money;
		}
		
		/**
		* @return age of customer
		*/
		int getAge(){
			return this->age;
		}
		
		
		/**
		* print all details of customer
		*/
		void print(){
			cout<<"Name: "<<this->name<<endl;
			cout<<"Operation: "<<this->operation<<endl;
			cout<<"Money: "<<this->money<<endl;
			cout<<"Age: "<<this->age<<endl;
		}
};
